components/dashboard/src/settings/EnvironmentVariables.tsx

```diff
@@ -79,6 +79,28 @@ function AddEnvVarModal(p: EnvVarModalProps) {
     </Modal>
 }
 
+function DeleteEnvVarModal(p: { variable: UserEnvVarValue, deleteVariable: () => void, onClose: () => void }) {
+    return <Modal visible={true} onClose={p.onClose}>
+        <h3 className="mb-4">Delete Variable?</h3>
+        <div className="border-t border-b border-gray-200 dark:border-gray-800 -mx-6 px-6 py-4 flex flex-col">
+            <div className="grid grid-cols-3 gap-4 px-3">
+                <h4 className="truncate">Name</h4>
```

Comment:
**suggestion**: What do you think of using the same font color and weight as the table headers in the environments list here? Comme ci comme ça?

Reply:
Good idea! Will try (FYI this is currently `text-sm text-gray-400` -- same in dark theme)